staging/src/k8s.io/controller-manager/go.mod

```diff
@@ -18,6 +18,7 @@ require (
 )
 
 replace (
+	github.com/spf13/viper => github.com/spf13/viper v1.7.0
```

Comment:
I don't think this is expected. You probably need to run the dependency lint script after fixing the other issues

Reply:
> I don't think this is expected. You probably need to run the dependency lint script after fixing the other issues

I've been running the scripts for the past few hours 😅. This is going to be a bit messy due to the viper upgrade.